common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -87,6 +87,35 @@ Here is an example from the same 5" quad with the harmonic notch configured:
     :target:  ../_images/post-tune-fft.png
     :width: 450px
 
+Step by step guide for Throttle-based Setup
+===========================================
+Based on: flat octocopter, Cube Orange, 6s Batteries, 16" Propellers, copter weight around 9kg.
+
+1. Check the parameters in Mission Planner and change them to:
+- Set :ref:`INS_LOG_BAT_MASK <INS_LOG_BAT_MASK>` = 1
+- Set :ref:`LOG_BITMASK <LOG_BITMASK>` (without RAW IMU)
+- Set :ref:`INS_HNTCH_ENABLE <INS_HNTCH_ENABLE>` = 1
+- Set :ref:`INS_LOG_BAT_MASK <INS_LOG_BAT_MASK>` = 7
+- Set :ref:`INS_LOG_BAT_OPT <INS_LOG_BAT_OPT>` = 1
```

Comment:
- Don't enable HNTCH parameters at this stage - you just want the data
- You have two values for BAT_MASK - use 7 as the default for 3 IMUs
- Best to give the actual LOG_BITMASK value
- LOG_BAT_OPT should be 0